mod_reforged/hooks/entity/tactical/humans/noble_arbalester.nut

```diff
@@ -37,11 +37,31 @@
 
 		if (this.m.Items.hasEmptySlot(::Const.ItemSlot.Body))
 		{
-			this.m.Items.equip(::new(::MSU.Class.WeightedContainer([
+			local armor = ::MSU.Class.WeightedContainer([
 				[1, "scripts/items/armor/gambeson"],
 				[1, "scripts/items/armor/padded_leather"],
 				[1, "scripts/items/armor/leather_lamellar"]
-			]).roll()));
+			]).roll();
+
+			if (armor != null)
```

Comment:
This null check is redundant as the weighted container above cannot return null.